### PR TITLE
Update information sub-perform02, sub-perform04

### PR DESCRIPTION
**Pull request by alexfoias**

### CODE REVIEW

File: participants.tsv [hunk 1]
```diff
@@ -164,9 +164,9 @@ sub-pavia04	F	25	-	-	2019-06-26	pavia	"Neuroradiology Unit, IRCCS Mondino Founda
 sub-pavia05	M	38	-	-	2019-06-26	pavia	"Neuroradiology Unit, IRCCS Mondino Foundation"	Siemens	Skyra	HeadNeck_20	syngo_MR_D13	"G. Savini, G. Germani, A. Pichiecchio"
 sub-pavia06	M	29	-	-	2019-07-03	pavia	"Neuroradiology Unit, IRCCS Mondino Foundation"	Siemens	Skyra	HeadNeck_20	syngo_MR_D13	"G. Savini, G. Germani, A. Pichiecchio"
 sub-perform01	F	23	176	63	2018-01-30	perform	"Concordia University, Perform Center, Montreal, Canada"	GE	MR750	-	27_LX_MR_Software_release:DV25.1_R02_1649.a	"J. Cohen-Adad, H. Benali"
-sub-perform02	M	27	-	-	2018-01-30	perform	"Concordia University, Perform Center, Montreal, Canada"	GE	MR750	-	27_LX_MR_Software_release:DV25.1_R02_1649.a	"J. Cohen-Adad, H. Benali"
+sub-perform02	M	27	-	83.91	2018-01-30	perform	"Concordia University, Perform Center, Montreal, Canada"	GE	MR750	-	27_LX_MR_Software_release:DV25.1_R02_1649.a	"J. Cohen-Adad, H. Benali"
```
Comment (jcohenadad) [inline participants.tsv]: ?

Comment (alexfoias) [reply participants.tsv]: that is what I found in the dicom metadata. we can round it if it is better.

Comment (jcohenadad) [reply participants.tsv]: ~are you sure we scanned a 84 y.o. person?~

my bad i thought that was the age 🤦‍♂️

**Review by jcohenadad — APPROVED** [review]

**Comment by jcohenadad** [comment]
@alexfoias can you please address this PR after https://github.com/spine-generic/data-multi-subject/pull/112

**Comment by alexfoias** [comment]
@jcohenadad sure.

**Comment by alexfoias** [comment]
@jcohenadad This is ready for review.

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**